### ISSUE TITLE
WCAG: Assess application using automated tools

### ISSUE THREAD

**Issue by dan-bowerman** [ramp4-pcar4/ramp-website#16]
A WCAG report on the state of compliance is updated to reflect results from an automated assessment of the application.

**Comment by spencerwahl** [comment]
Ran the `WAVE Evalutation Tool` chrome extension as well as a contrast checker on the site.

Issues found (on visible elements):
- missing alt-text on images.


Contrast was deemed good from both tools.

**Comment by spencerwahl** [comment]
After #33 the automated checkers only flag missing alt-text on a basemap tile in RAMP, and some issues on invisible elements also in RAMP.
